docs/source/quickstart.mdx

```diff
@@ -0,0 +1,545 @@
+If this composition succeeds, GraphOS updates your router with the result.
+
+In this step, you'll create an empty supergraph in GraphOS Studio and then publish subgraph schemas to it using Rover.
+
+#### Create supergraph
+
+1. Go to [studio.apollographql.com](https://studio.apollographql.com/?referrer=docs-content) and click **+ Create New Graph**.
+2. Enter a **Graph title**, for example `Federation Quickstart`. Leave the **Graph Architecture** as `Supergraph (Default)` and **Visible to Team Members** as `On`. Click **Next**.
+3. A **Publish your schema using Federation** dialog appears. Copy the example code block to a local text file.
+
+    <img
+      src="./images/quickstart/publish-your-schema.jpg"
+      className="screenshot"
+      width="500"
+      alt="A dialog from GraphOS Studio displays a hidden API key and graph ref."
+    />
+
+    - The protected value that appears after `APOLLO_KEY=` and beginning with `service:` is your graph API key.
+    - The value that appears after `rover subgraph publish` is your graph ref. By default, it ends with `@current`.
+
+    <Caution>
+
+      API keys are secret credentials. Never share them outside your organization or commit them to version control. Delete and replace API keys that you believe are compromised.
+
+    </Caution>
+
+4. Leave the dialog open; you'll complete setup with Rover.
+
+#### Publish subgraph schemas
+
+1. Ensure your subgraphs are still [running locally](#run-subgraphs).
+2. Take the example code block you copied from Studio and replace the last three lines with the following:
+
+    ```bash showLineNumbers=false
+        --schema ./subgraphs/products/schema.graphql \
+        --name products \
+        --routing-url http://localhost:4001/products/graphql
+    ```
+
+    It should look something like the following, with the `<GRAPH_API_KEY>` and `<GRAPH_REF>` replaced with your graph's values:
+
+    ```bash showLineNumbers=false
+    APOLLO_KEY=service:<GRAPH_API_KEY> \
+    rover subgraph publish <GRAPH_REF> \
+    --schema ./subgraphs/products/schema.graphql \
+    --name products \
+    --routing-url http://localhost:4001/products/graphql
+    ```
+
+    - The `--schema` option specifies the path to the schema file to publish. 
+    - The `--name` option specifies the name of the subgraph.
+    - The `--routing-url` is the address the router should send requests to.
+    
+    <Note>
+
+    In a production supergraph, a subgraph's routing URL is generally its public endpoint.
+
+    </Note>
+
+2. In a new terminal window, paste and run your multi-line command.
+    - Because the command provides a `localhost` address for the `routing-url`, the terminal will confirm that you want to publish. Enter `Y`.
+    - If the command is successful, you'll see the following in your terminal:
+
+      ```bash showLineNumbers=false disableCopy=true
```

Comment:
Same weird highlighting of some keywords, maybe bc of the bash languages of the Codeblock?